bin/yaml/libraries.yaml

```diff
@@ -154,6 +154,19 @@ libraries:
       - 1.0.0
       - 1.1.5
       type: github
+    copperspice:
+      build_type: none
+      check_file: include/cs-config.h
+      compression: bz2
+      create_untar_dir: true
+      dir: libs/copperspice/{{name}}
+      lib_type: shared
+      targets:
+      - 1.8.0
+      type: tarballs
+      untar_dir:
+        name: null
```

Comment:
Oh weird - was this needed to fix up something awful in the installer (presumably it defaults to something unless you define it as `null` here?)

Reply:
oh... no... this is wrong i think.. i did a `bin/ce_install reformat` before committing and didn't check.. will see what's up

Reply:
Ok, I found out that if you put {name} in there, it reformats to this. And putting {{name}} will give a parsing error. The trick apparently is to put it in quotes.